gr-uhd/lib/usrp_block_impl.h

```diff
@@ -165,16 +164,16 @@ class usrp_block_impl : virtual public usrp_block
     // - If chan is -1, it depends on minus_one_updates_all:
     //   - Either set vector_to_update[0] or
     //   - Set *all* entries in vector_to_update
-    // - Returns a dynamic_bitset, all indexes that where changed in
-    //   vector_to_update are set to 1
+    // - Returns a bool vector, all indexes that where changed in
+    //   vector_to_update are set to true
     template <typename T>
-    static boost::dynamic_bitset<>
+    static std::vector<bool>
     _update_vector_from_cmd_val(std::vector<T>& vector_to_update,
                                 int chan,
                                 const T cmd_val,
                                 bool minus_one_updates_all = false)
```

Comment:
During testing I noticed that this function is unused. (The last usage was removed in 41bde7a0209d4b0c682d06db656d092062ad943c.) So I'll just delete it.